sumatra/datastore/archivingfs.py

```diff
@@ -17,7 +17,7 @@
 from sumatra.core import TIMESTAMP_FORMAT, registry
 
 
-from .base import DataItem, DataKey
```

Comment:
Unless I'm mistaken, `DataKey` needs to be available in the `archivingfs` namespace.